cob_default_robot_config/cob4-2/head_joint_configurations.yaml

```diff
@@ -5,16 +5,16 @@ default_vel: 0.4
 
 home: [[0.0, 0.0, 0.0]]
 
-front: [[0.0, 0.0, 0.0]]
-front_down: [[1.57, -3.14, 1.57]]
-front_down_twisted: [[1.57, -3.14, -1.57]]
+front: [[0.0, 0.0, 3.14]]
```

Comment:
The problem is solved after update the URDF model , see https://github.com/ipa320/cob_common/pull/189 
These changes are already undo https://github.com/ipa320/cob_robots/pull/473/commits/d25d82cf03d900d2f59e92d08f82e0a0dcbeaac0 on the https://github.com/ipa320/cob_robots/pull/473